File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/billing/shared/dto/opd-serviceitem-price.dto.ts
export class OpdServiceItemPrice_DTO {
    public DepartmentId: number = 0;
    public DepartmentName: string = null;
    public PerformerId: number = 0;
    public PerformerName: string = null;
    public ServiceDepartmentId: number = 0;
    public ServiceDepartmentName
    public ServiceItemId: number = 0;
    public ItemCode: string = null;
    public ItemName: string = null;
    public PriceCategoryId: number = 0;
    public Price: number = 0;
    public IsTaxApplicable: boolean = false;
    public IsZeroPriceAllowed: boolean = false;
    public IsPriceChangeAllowed: boolean = false;
    public IsDiscountApplicable: boolean = false;
}

File: Code/Websites/DanpheEMR/Services/Billing/DTO/OpdServiceItemPrice_DTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DanpheEMR.Services.Billing.DTO
{
    public class OpdServiceItemPrice_DTO
    {
        public int DepartmentId { get; set; }
        public string DepartmentName { get; set; }
        public int PerformerId { get; set; }
        public string PerformerName { get; set; }

        public int ServiceDepartmentId { get; set; }
        public string ServiceDepartmentName { get; set; }
        public int ServiceItemId { get; set; }
        public string ItemCode { get; set; }
        public string ItemName { get; set; }
        public int PriceCategoryId { get; set; }
        public decimal Price { get; set; }
        public bool IsTaxApplicable { get; set; }
        public bool IsZeroPriceAllowed { get; set; }
        public bool IsPriceChangeAllowed { get; set; }
            
        public bool IsCoPayment { get; set; }
        public decimal CoPaymentCashPercent { get; set; }
        public decimal CoPaymentCreditPercent { get; set; }
        public decimal DiscountPercent { get; set; }

        public bool IsDiscountApplicable { get; set; }
    }
}
